### PR TITLE
#299F Update templatePermission allowedSections outside restriction

### PR DESCRIPTION
**Pull request by nmadruga**
Front-end branch to update types and query for [Back-end#299](https://github.com/openmsupply/application-manager-server/issue/299)

### Back-end branch PR
[#299](https://github.com/openmsupply/application-manager-server/pull/358)

### Changes
- Update generated types
- Fix queries to use the new field allowedSections on reviewAssignment

### CODE REVIEW

File: src/utils/graphql/queries/getReviewInfo.query.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default gql`
         levelNumber
         reviewerId
         isLastLevel
-        templateSectionRestrictions
+        allowedSections
```
Comment (CarlosNZ) [inline src/utils/graphql/queries/getReviewInfo.query.ts:15]: Do we also want to get `canSelfAssign` and `restrictions` here as well?

**Review by CarlosNZ — COMMENTED** [review]
Looks good, thanks.

Just wondering -- where is the `canSelfAssign` value being retrieved from currently? Do we need update that too?

File: src/utils/graphql/queries/getReviewInfo.query.ts (hunk 1 above)
Comment (nmadruga) [reply src/utils/graphql/queries/getReviewInfo.query.ts:15]: The only field present in `reviewAssignment` is `allowedSections`. The other mentioned fields are part of `templatePermission` and used for generating the next assignments based on `restrictions` and if the reviewer to be assigned `canSelfAssign`.

**Review by CarlosNZ — APPROVED** [review]
LGTM :)

**Comment by nmadruga** [comment]
Bah -- merged `master` in and somehow that wasn't good. Will have to created a new branch without this merge and work from there.

**Pull request closed without merging**